vizro-core/schemas/generate.py

```diff
@@ -17,7 +17,10 @@
 
 if args.check:
     if json.loads(schema_path.read_text()) != json.loads(schema_json):
-        sys.exit("JSON schema is out of date. Run `hatch run schema` to update it.")
+        # Ideally just doing hatch run:schema would be fine, but the schema slightly depends
+        # on Python version (Python 3.8 vs. Python 3.11 give different results), even
+        # for the same pydantic version.
+        sys.exit("JSON schema is out of date. Run `hatch run all.py3.11:schema` to update it.")
```

Comment:
Can we not do a double aliasing so that `hatch run schema` would actually run `hatch run all.py3.11:schema`. That seems easier no?

Reply:
We could indeed do this. It's kind of ugly because doing `hatch run schema` then actually creates both the `default` environment and then the `all.py3.11` inside that environment. So it's a pretty nasty hack but it does seem to work ok for `hatch run example`, so let me change it here - good idea!

Reply:
Ah, also the reason I didn't do this before was because `hatch prep-release` runs from `default` environment, and so it would have been much slower to run that because it would have needed to create the `all.py3.11` environment half way through that. But now that environment creation is so fast this is all good 🎉